docs/Examples.md

```diff
@@ -459,8 +459,8 @@ spec:
 ```
 
 
-### Example: Cluster Master
-In the ClusterMaster example, app3 and app4 are installed on any indexer instances that are managed by the cluster master. App5 and app6 are installed locally on the ClusterMaster instance.
+### Example: Cluster Manager
+In the ClusterMaster example, app3 and app4 are installed on any indexer instances that are managed by the cluster manager. App5 and app6 are installed locally on the ClusterMaster instance.
```

Comment:
Let's just use "In this example, app3...."

Reply:
Hi @eddikay.  If I understood correctly you're suggesting to replace:

`In the ClusterMaster example, app3 ...`

with 

`In this example, app3 ...`

Reply:
Yes! Thank you :)

Reply:
done.